Source/Resources/Content/MSGraph/DeviceManagement/DeviceManagementScripts/OS - Set-TimeZone.json

```diff
@@ -7,7 +7,7 @@
       }
     }
   ],
-  "description": "Configures time zone to be set automatically.",
+  "description": "What is it: Configures time zone to be set automatically. \n Simeon Notes: Some computers have issues using Auto Time Zone Updater because of ISP or network restrictions; this script sets the default time zone just in case.",
```

Comment:
**What does this do?**
Configures Windows to set time zone automatically based on location. Also sets the default computer time zone based on public IP address, as some computers have issues using the Auto Time Zone Updater because of ISP or network restrictions.

**Why should you use this?**
To automatically set the time zone based on location.

**What is the end-user impact?**
The time zone will be set automatically based on the device location.

**Learn more**
N/A